main.tf

```diff
@@ -34,7 +44,15 @@ module "obs_staging_vpc" {
   xchain_settings = {
     enabled      = true
     docker_image = local.xchain_indexer_staging_docker_image
-    config       = "staging"
+    config       = "config"
+    json_config = jsonencode({
```

Comment:
Not sure if we even need a templatefile here, jsonencode does the trick. Downside is that we need to repeat this in two places, so having a template file might prove beneficial if the config structure will be subject to frequent changes in the future